docs/quickstart.rst

```diff
@@ -0,0 +1,405 @@
+.. _Privacy Policy: https://www.refine.bio/privacy
+
+To create a token, make an object of the Token class:
+
+.. code-block:: python
+
+    import pyrefinebio
+
+    token = pyrefinebio.Token()
+
+Then, to activate your token, call `agree_to_terms_and_conditions()` on it:
+
+.. code-block:: python
+
+    token.agree_to_terms_and_conditions()
+
+Now, your Token is fully set up and will be used in any API requests made during the execution of this script.
+
+If you want to use the same Token in future scripts, you can save it to the Config file:
+
+.. code-block:: python
+
+    token.save_token()
+
+Now, the token will be automatically loaded every time you import pyrefinebio.
+
+If you want to get access to the token that is saved to the config file in the future, you can load it:
+
+.. code-block:: python
+
+    token = pyrefinebio.Token.load_token()
+
+For more information check out the :ref:`Token` class documentation and :ref:`Config`.
+
+Downloading Datasets
+--------------------
+
+After you set up and activate a Token you can start creating and downloading Datasets.
+
+See :ref:`Quickstart/Setting up Tokens` for a tutorial on setting up a Token.
+
+pyrefinebio provides the function `download_dataset()` for creating and downloading Datasets.
+It will will automatically handle every part of the creation and download process for you.
+You will receive the Dataset as a zip file.
+
+`download_dataset()` requires that you pass in the parameters `path`, `email_address`, and either `dataset_dict` or `experiments`.
+
+* `path` is the path that the Dataset will be downloaded to. You specify a path to a zip file or a directory. If you pass in a path to a directory, the name of the zip file will be automatically generated in the format `dataset-<dataset_id>.zip`.
+
+* `email_address` is the email address that will be notified when the Dataset is finished processing.
+
+The parameters `experiments` and `dataset_dict` both control the which Experiments and Samples will be a part of the Dataset.
+
+* `experiments` can be used when you want to add specific Experiments to your Dataset. All the downloadable samples associated with the Experiments that you pass in will be added to the Dataset. 
+
+The `experiments` parameter is just a list of Experiment accession codes or pyrefinebio Experiment objects. Here's an example:
+
+.. code-block:: python
+
+    ex2 = pyrefinebio.Experiment.get("<Experiment 2 Accession Code>")
+
+    exs = ["<Experiment 1 Accession Code>", ex2]
+
+* `dataset_dict` should be used when you want to specify specific Samples to be included in the Dataset. It should be in the format:
+
+.. code-block:: python
+
+    dd = {
+        "<Experiment 1 Accession Code>": [
+            "<Sample 1 Accession Code>",
+            "<Sample 2 Accession Code>"
+        ],
+        "<Experiment 2 Accession Code>": [
+            "<Sample 3 Accession Code>",
+            "<Sample 4 Accession code>"
+        ]
+    }
```

Comment:
You should also mention the `"<Experiment Accession Code>": ["ALL"]` Trick here.